wave_utils/src/math.cpp

```diff
@@ -86,4 +86,12 @@ double wrapTo360(double euler_angle) {
     }
 }
 
+Mat3 skew_symmetric_matrix(Vec3 a){
+  Mat3 skew_matrix;
+  skew_matrix << 0, -a(2), a(1),
```

Comment:
I don't really see it as a big problem actually, because:
- the directive is short and well defined 
- I intend to ask people to use the clang-format tool to help speed up formatting (common format without question)
- clang-format is IDE-agnostic
- removing clang-format directives is literally 2 commands away 
```
    cat file.cpp | '/\/\/\ clang\-format\ off/,+1 d'
    cat file.cpp | '/\/\/\ clang\-format\ on/,+1 d'
```